### PR TITLE
Output edits: WIP Do not merge

### PR DESCRIPTION
**Pull request by hlucey**
This is a work in progress. Do not merge yet, thanks!

### CODE REVIEW

**Comment by hlucey** [comment]
Variable.tf comments: 

1. Can we make sure that all the variables of the same name use the updated description

2. The descriptions for the input variables don't offer much information and might be more useful if they had links to the docs for the underlying services. For example, the tag description,should we link to the watsonx docs that give some context about tags in watsonx projects or here: https://dataplatform.cloud.ibm.com/docs/content/wsj/getting-started/assets.html?context=wx&audience=wdp#prop-data

3. Are the limitations or additional context for any of the values? For example... "The list can be up to 255 characters and can contain spaces, letters, numbers, underscores, dashes, and the symbols # and @."

4. Variable "ibmcloud_api_key" description = "The API key that's used with the Terraform IBM Cloud and IBM provider."
What does this description mean…?

5. Variable "resource_prefix":  Can you explain what this is? The description is a little confusing.

File: ibm_catalog.json [hunk 1]
```diff
@@ -277,123 +277,123 @@
           "outputs": [
             {
               "key": "watson_platform_endpoint",
-              "description": "Endpoint of the Watson platform"
+              "description": "Endpoint of the Watson platform."
             },
             {
               "key": "watsonx_assistant_crn",
-              "description": "ID of the WatsonX Assistant instance"
+              "description": "ID of the WatsonX Assistant instance."
```
Comment (mkrudele) [inline ibm_catalog.json:284]: Should be `The CRN of the ...`

File: ibm_catalog.json [hunk 2]
```diff
@@ -277,123 +277,123 @@
...
             },
             {
               "key": "watsonx_assistant_guid",
-              "description": "ID of the WatsonX Assistant instance"
+              "description": "ID of the WatsonX Assistant instance."
             },
             {
               "key": "watsonx_assistant_name",
-              "description": "ID of the WatsonX Assistant instance"
+              "description": "ID of the WatsonX Assistant instance."
```
Comment (mkrudele) [inline ibm_catalog.json:292]: Should be `The name of the...`

File: ibm_catalog.json [hunk 3]
```diff
@@ -277,123 +277,123 @@
...
             },
             {
               "key": "watsonx_assistant_plan_id",
-              "description": "ID of the WatsonX Assistant instance"
+              "description": "ID of the WatsonX Assistant instance."
```
Comment (mkrudele) [inline ibm_catalog.json:296]: Should be `The ID of the watson... service plan`

File: ibm_catalog.json [hunk 4]
```diff
@@ -277,123 +277,123 @@
...
             },
             {
               "key": "watsonx_assistant_dashboard_url",
-              "description": "Dashboard URL of the WatsonX Assistant instance"
+              "description": "Dashboard URL of the WatsonX Assistant instance."
             },
             {
               "key": "watsonx_discovery_crn",
-              "description": "ID of the WatsonX Discovery instance"
+              "description": "ID of the WatsonX Discovery instance."
```
Comment (mkrudele) [inline ibm_catalog.json:304]: Should be `The CRN of..`

File: ibm_catalog.json [hunk 5]
```diff
@@ -277,123 +277,123 @@
...
             },
             {
               "key": "watsonx_discovery_guid",
-              "description": "ID of the WatsonX Discovery instance"
+              "description": "ID of the WatsonX Discovery instance."
             },
             {
               "key": "watsonx_discovery_name",
-              "description": "ID of the WatsonX Discovery instance"
+              "description": "ID of the WatsonX Discovery instance."
```
Comment (mkrudele) [inline ibm_catalog.json:312]: Should be `the name of ...`

File: ibm_catalog.json [hunk 6]
```diff
@@ -277,123 +277,123 @@
...
             },
             {
               "key": "watsonx_discovery_plan_id",
-              "description": "ID of the WatsonX Discovery instance"
+              "description": "ID of the WatsonX Discovery instance."
```
Comment (mkrudele) [inline ibm_catalog.json:316]: Should be `the watsonx... service plan id`

File: ibm_catalog.json [hunk 7]
```diff
@@ -277,123 +277,123 @@
...
             },
             {
               "key": "watsonx_discovery_dashboard_url",
-              "description": "Dashboard URL of the WatsonX Discovery instance"
+              "description": "Dashboard URL of the WatsonX Discovery instance."
             },
             {
               "key": "watson_machine_learning_crn",
-              "description": "ID of the Watson Machine Learning instance"
+              "description": "ID of the Watson Machine Learning instance."
```
Comment (mkrudele) [inline ibm_catalog.json:324]: Should be `the CRN of ...`

File: ibm_catalog.json [hunk 8]
```diff
@@ -277,123 +277,123 @@
...
             },
             {
               "key": "watson_machine_learning_guid",
-              "description": "ID of the Watson Machine Learning instance"
+              "description": "ID of the Watson Machine Learning instance."
             },
             {
               "key": "watson_machine_learning_name",
-              "description": "ID of the Watson Machine Learning instance"
+              "description": "ID of the Watson Machine Learning instance."
```
Comment (mkrudele) [inline ibm_catalog.json:332]: Should be `the name of ...`

File: ibm_catalog.json [hunk 9]
```diff
@@ -277,123 +277,123 @@
...
             },
             {
               "key": "watson_machine_learning_plan_id",
-              "description": "ID of the Watson Machine Learning instance"
+              "description": "ID of the Watson Machine Learning instance."
```
Comment (mkrudele) [inline ibm_catalog.json:336]: Should be `the watson ... service plan id`

File: ibm_catalog.json [hunk 10]
```diff
@@ -277,123 +277,123 @@
...
             },
             {
               "key": "watson_machine_learning_dashboard_url",
-              "description": "Dashboard URL of the Watson Machine Learning instance"
+              "description": "Dashboard URL of the Watson Machine Learning instance."
             },
             {
               "key": "watson_studio_crn",
-              "description": "ID of the Watson Studio instance"
+              "description": "ID of the Watson Studio instance."
```
Comment (mkrudele) [inline ibm_catalog.json:344]: Should be `the CRN of ...`

File: ibm_catalog.json [hunk 11]
```diff
@@ -277,123 +277,123 @@
...
             },
             {
               "key": "watson_studio_guid",
-              "description": "ID of the Watson Studio instance"
+              "description": "ID of the Watson Studio instance."
             },
             {
               "key": "watson_studio_name",
-              "description": "ID of the Watson Studio instance"
+              "description": "ID of the Watson Studio instance."
```
Comment (mkrudele) [inline ibm_catalog.json:352]: Shuld be `the name of ...`

File: ibm_catalog.json [hunk 12]
```diff
@@ -277,123 +277,123 @@
...
             },
             {
               "key": "watson_studio_plan_id",
-              "description": "ID of the Watson Studio instance"
+              "description": "ID of the Watson Studio instance."
```
Comment (mkrudele) [inline ibm_catalog.json:356]: Should be `the watson studio.. service plan id`

File: ibm_catalog.json [hunk 13]
```diff
@@ -277,123 +277,123 @@
...
             },
             {
               "key": "watson_studio_dashboard_url",
-              "description": "Dashboard URL of the Watson Studio instance"
+              "description": "Dashboard URL of the Watson Studio instance."
             },
             {
               "key": "watson_governance_crn",
-              "description": "ID of the Watson Governance instance"
+              "description": "ID of the Watson Governance instance."
```
Comment (mkrudele) [inline ibm_catalog.json:364]: Should be `the CRN of ...`

File: ibm_catalog.json [hunk 14]
```diff
@@ -277,123 +277,123 @@
...
             },
             {
               "key": "watson_governance_guid",
-              "description": "ID of the Watson Governance instance"
+              "description": "ID of the Watson Governance instance."
             },
             {
               "key": "watson_governance_name",
-              "description": "ID of the Watson Governance instance"
+              "description": "ID of the Watson Governance instance."
```
Comment (mkrudele) [inline ibm_catalog.json:372]: Should be `the name of...`

File: ibm_catalog.json [hunk 15]
```diff
@@ -277,123 +277,123 @@
...
             },
             {
               "key": "watson_governance_plan_id",
-              "description": "ID of the Watson Governance instance"
+              "description": "ID of the Watson Governance instance."
```
Comment (mkrudele) [inline ibm_catalog.json:376]: Should be `the watson gove... service plan id`

File: ibm_catalog.json [hunk 16]
```diff
@@ -277,123 +277,123 @@
...
             },
             {
               "key": "watson_governance_dashboard_url",
-              "description": "Dashboard URL of the Watson Governance instance"
+              "description": "Dashboard URL of the Watson Governance instance."
             },
             {
               "key": "watsonx_project_id",
-              "description": "ID of the Watson project to create"
+              "description": "ID of the Watson project."
```
Comment (mkrudele) [inline ibm_catalog.json:384]: The ID of the Watsonx project

File: ibm_catalog.json [hunk 17]
```diff
@@ -277,123 +277,123 @@
...
             },
             {
               "key": "watsonx_project_location",
-              "description": "Location of the Watson project to create"
+              "description": "Location of the Watson project."
```
Comment (mkrudele) [inline ibm_catalog.json:388]: watsonx project

File: ibm_catalog.json [hunk 18]
```diff
@@ -277,123 +277,123 @@
...
             },
             {
               "key": "watsonx_project_url",
-              "description": "URL of the Watson project to create"
+              "description": "URL of the Watson project."
```
Comment (mkrudele) [inline ibm_catalog.json:392]: watsonx project

File: outputs.tf [hunk 19]
```diff
@@ -1,149 +1,149 @@
 output "watson_platform_endpoint" {
-  description = "Endpoint of the Watson platform"
+  description = "The endpoint of the Watson platform."
```
Comment (mkrudele) [inline outputs.tf:2]: watsonx platform

File: variables.tf [hunk 20]
```diff
@@ -1,23 +1,23 @@
 variable "ibmcloud_api_key" {
-  description = "Used with the Terraform IBM-Cloud/ibm provider."
+  description = "The API key that's used with the IBM Cloud Terraform and IBM provider."
```
Comment (mkrudele) [inline variables.tf:2]: Should be `the IBM Cloud Terraform IBM provider`

File: variables.tf [hunk 21]
```diff
@@ -1,23 +1,23 @@
...
   sensitive   = true
   type        = string
 }
 
 variable "watsonx_admin_api_key" {
   default     = null
-  description = "Used to call Watson APIs to configure the user and the project."
+  description = "The API key that's used to call Watson APIs to configure the user and the project."
```
Comment (mkrudele) [inline variables.tf:9]: We should provide a more user friendly description. Remove reference to an api call. Something like:  
`The API key of the IBM watsonx admin in the target account`

File: variables.tf [hunk 22]
```diff
@@ -1,23 +1,23 @@
...
   sensitive   = true
   type        = string
 }
 
 variable "location" {
   default     = "us-south"
-  description = "Used with the Terraform IBM-Cloud/ibm provider as well as resource creation."
+  description = "The location that's used with the IBM Cloud Terraform and IBM provider. It's also used during resource creation."
```
Comment (mkrudele) [inline variables.tf:16]: Should be `IBM Cloud Terraform IBM provider`

**Review by mkrudele — CHANGES_REQUESTED** [review]

**Review by mkrudele — APPROVED** [review]

**Review by andreainnocenti — APPROVED** [review]

**Pull request merged**